source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -188,31 +189,33 @@
     'MINUTES': 2880,
 }
 
+# Email service configuration (using Anymail).
+# Refer to Anymail's documentation for configuration details.
+
+ANYMAIL = {
+    "SENDINBLUE_API_KEY": "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",
+    'TEMPLATES': {
+        "CONFIRM_SIGN_UP": "example_template_id",
+        "FORGOT_PASSWORD": "example_template_id",
+    },
+}
+EMAIL_BACKEND = 'anymail.backends.sendinblue.EmailBackend'
+# This 'FROM' email is not used with SendInBlue templates
+DEFAULT_FROM_EMAIL = 'noreply@example.org'
+
 # These settings are not related to the core API functionality. Feel free to
 # edit them to your needs.
 # NOTE: "{{token}}" is a placeholder for the real activation token. It will be
 #       dynamically replaced by the real "token".
 CONSTANT = {
-    "EMAIL_SERVICE": False,
+    "EMAIL_SERVICE": True,
```

Comment:
Please, do not change default configuration of server. Server admin can have big problem if their overwrite settings doesn't overwrite ALL settings.